doc/source/user_guide/advanced.rst

```diff
@@ -179,18 +179,18 @@ on a deeper level.
 
 .. _advanced.shown_levels:
 
-Defined Levels
+Defined levels
 ~~~~~~~~~~~~~~
 
-The :class:`MultiIndex` keeps all the defined levels of an index, even
+The repr of a ``MultiIndex`` shows all the defined levels of an index, even
 if they are not actually used. When slicing an index, you may notice this.
 For example:
 
 .. ipython:: python
 
-   df.columns.levels  # original MultiIndex
+   df.columns  # original MultiIndex
```

Comment:
Can you revert this? 
I would keep it here strictly on capitalisation changes, that makes it easier to review.

I think the above is also not a correct change, as we just changed the repr of a MI, and it will no longer show all levels, so you really need to look at the `.levels` attribute